binding.gyp

```diff
@@ -21,7 +21,7 @@
         }],
       ],
       "defines": [
-        'NODEREPORT_VERSION="<!(node -p \"require(\'./package.json\').version\")"'
+        'node-report_VERSION="<!(node -p \"require(\'./package.json\').version\")"'
```

Comment:
Please either revert this, or upper case. I don't believe `-` is a valid character for C/C++ macro names.